src/WebJobs.Script/Description/NodeFunctionInvoker.cs

```diff
@@ -55,6 +58,8 @@ internal NodeFunctionInvoker(ScriptHost host, BindingMetadata trigger, FunctionM
             _script = string.Format(CultureInfo.InvariantCulture, _functionTemplate, scriptFilePath);
             _inputBindings = inputBindings;
             _outputBindings = outputBindings;
+            _functionMetadata = functionMetadata;
```

Comment:
Same as with the C# invoker, we should be able to use the property exposed by the base type.